database/migrations/2021_08_15_070139_create_facilities_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateFacilitiesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('facilities', function (Blueprint $table) {
+            $table->id();
+            $table->string('picture');
```

Comment:
[SHOULD]
nullを許可してあげるカラムには下記のようなnullableを付与してあげると良さそうです！
```suggestion
            $table->string('picture')->nullable();
```